### ISSUE TITLE
Error parsing LocalDate

### ISSUE THREAD

**Issue by anne-nelson** [gekomad/itto-csv#4]
I'm having an issue trying to parse a `java.time.LocalDate` from a file, but the strange thing is that I was able to parse `java.time.LocalDatetime` fine.

This compiles
```
import java.time.LocalDateTime

import cats.data.NonEmptyList
import com.github.gekomad.ittocsv.core.ParseFailure
import com.github.gekomad.ittocsv.parser.IttoCSVFormat
import com.github.gekomad.ittocsv.core.FromCsv._

case class CustomerId(value: String)
case class Customer(id: CustomerId, date: LocalDateTime)

object FileParser {
  implicit val csvFormat = IttoCSVFormat.default

  implicit def csvFieldToId: String => Either[ParseFailure, CustomerId] = (s => Right(CustomerId(s)))

  implicit def csvFieldToLocalDateTime: String => Either[ParseFailure, LocalDateTime] = {
    s =>
      scala.util.Try {
        Right(LocalDateTime.parse(s.trim))
      }.getOrElse(Left(ParseFailure(s"Could not convert '$s' to LocalDate")))
  }

  def parseFile(file: String): List[Either[NonEmptyList[ParseFailure], Customer]] = {
    val fileWithoutHeader = file.split("\n").drop(1).toList
    fromCsv[Customer](fileWithoutHeader).toList
  }
}

```

But this (which is the code above but with `LocalDateTime` replaced with `LocalDate` everywhere) does not compile
```
import java.time.LocalDate

import cats.data.NonEmptyList
import com.github.gekomad.ittocsv.core.ParseFailure
import com.github.gekomad.ittocsv.parser.IttoCSVFormat
import com.github.gekomad.ittocsv.core.FromCsv._

case class CustomerId(value: String)
case class Customer(id: CustomerId, date: LocalDate)

object FileParser {
  implicit val csvFormat = IttoCSVFormat.default

  implicit def csvFieldToId: String => Either[ParseFailure, CustomerId] = (s => Right(CustomerId(s)))

  implicit def csvFieldToLocalDate: String => Either[ParseFailure, LocalDate] = {
    s =>
      scala.util.Try {
        Right(LocalDate.parse(s.trim))
      }.getOrElse(Left(ParseFailure(s"Could not convert '$s' to LocalDate")))
  }

  def parseFile(file: String): List[Either[NonEmptyList[ParseFailure], Customer]] = {
    val fileWithoutHeader = file.split("\n").drop(1).toList
    fromCsv[Customer](fileWithoutHeader).toList
  }
}
```

Trying to compile the one with `LocalDate` gives the error below, which as far as I know means that I haven't provided an implicit to parse one of the fields in my case class, but I've provided a parser for both.
```
[error]     could not find implicit value for evidence parameter of type com.github.gekomad.ittocsv.core.Schema[entities.Customer]
[error]     fromCsv[Customer](fileWithoutHeader).toList
```

Should I be able to parse any random class? I'm using v1.1.0

**Comment by gekomad** [comment]
Thank you for pointing, I've released just now the version 1.1.1 on mvnrepository, it gets some time to publish. To use immediatly the library, add `resolvers ++= Seq(  "Sonatype Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots")` to your buidl.sbt

**Comment by anne-nelson** [comment]
Thank you very much for the quick response, it's compiling now 😊
